pipelines/short-read-alignment-vector/ShortReadAlignment.wdl

```diff
@@ -366,18 +366,18 @@ task FixMateInformation {
   Int disk_size = (ceil(size(input_file, "GiB")) * 3) + 20
 
   command {
-    java -Xms7000m -jar /bin/picard.jar \
+    java -Xmx7000m -jar /bin/picard.jar \
       FixMateInformation \
       INPUT=~{input_file} \
       OUTPUT=~{output_bam_basename}.bam \
       MAX_RECORDS_IN_RAM=300000 \
       CREATE_INDEX=true
   }
   runtime {
-    docker: runTimeSettings.picard_docker
-    preemptible: runTimeSettings.preemptible_tries
-    disks: "local-disk " + disk_size + " HDD"
-    memory: "7.5 GiB"
+    singularity: runTimeSettings.picard_singularity_image
+    memory: 4000
```

Comment:
Is this in MB? If so should the Xmx value be similar to this value?

Reply:
Yes, that's right.  I've adjusted the memory settings.